src/DotLiquid/StandardFilters.cs

```diff
@@ -359,6 +359,9 @@ public static string Rstrip(string input)
         /// <seealso href="https://shopify.dev/api/liquid/filters/money-filters#money">Shopify Money filter</seealso>
         public static string Currency(Context context, object input, string languageTag = null)
         {
+            // Check for null input, return null
+            if (input == null) return null;

```

Comment:
```suggestion
            if (string.IsNullOrEmpty(input))
                return input;
```

Reply:
`string.IsNullOrEmpty(input)` is not working unless I explicitly cast the input(which is an object) to a string. and that explicit cast is not working with other datatypes and causing some existing testcases to fail.

But I updated the test case to handle the empty string scenario as well. the empty string scenario was working as expected even before this change.

Reply:
Thanks for that.